### PR TITLE
More EDB examples

### PR DESCRIPTION
**Pull request by aladshaw3**
## Fixes/Addresses:

Creation of additional usage examples for EDB and moving those examples to a proper location.

## Summary/Motivation:

In order for EDB to start seeing usage, we need to have a set of examples that user's can view and build from. This PR establishes a directory for those specific examples to live and tests them with the MockDB setup. 

## Changes proposed in this PR:
- New folder under watertap/examples for edb examples
- Tests with MockDB

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by andrewlee94 — DISMISSED** [review]
I don't see anything wrong.

File: watertap/examples/edb/simple_acid.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+            H2CO3 <--> H_+ + HCO3_-
+            HCO3_- <--> H_+ + CO3_2-
+
+
+    (8) When using an reactor object in IDAES, you must always provide a 'reaction_config'
+        to match with the 'thermo_config'. We can create a base 'reaction' config from
+        the database and add reactions to that config in the same way we do for the
+        'thermo_config' when adding reactions as inherent.
+
+        [NOTE: If a reaction is added to a 'thermo_config' as 'inherent', this it should
```
Comment (adam-a-a) [inline watertap/examples/edb/simple_acid.py]:
```suggestion
        [NOTE: If a reaction is added to a 'thermo_config' as 'inherent', it should
```

**Review by srikanthallu — DISMISSED** [review]

**Review by adam-a-a — DISMISSED** [review]
LGTM

**Review by andrewlee94 — APPROVED** [review]
Looks good to me.

**Comment by adam-a-a** [comment]
@andrewlee94, @srikanthallu Can you reapprove?

**Review by adam-a-a — APPROVED** [review]
LGTM

**Review by srikanthallu — APPROVED** [review]

**Pull request merged**